src/user.spec.js

```diff
@@ -217,6 +218,74 @@ describe("user", () => {
     });
   });
 
+  describe(".changePassword()", () => {
+
+    test.skip("change the actual user password", async () => {
```

Comment:
If you remove the `.skip` and test : 
the error will be `JsonRpcError { code: 3, data: undefined }`

@badrAZ

Reply:
it's not the `getConnection` which breaks it's the next `xo.call`